### ISSUE TITLE
Only get 60 mAP VGG model

### ISSUE THREAD

**Issue by samuel1208** [balancap/SSD-Tensorflow#48]
I only get 60 mAP on vgg model. The vgg model is download from [here](https://drive.google.com/file/d/0B0qPCUZ-3YwWZlJaRTRRQWRFYXM/view).  My Evaluation command is like following:

```
python eval_ssd_network.py --eval_dir=./datatmp/ --dataset_dir=./datatmp/voc07_test.tfrecord --dateset_name=\
    voc07 --model_name=ssd_300_vgg  --checkpoint_path=./datatmp/VGG_VOC0712_SSD_300x300_iter_120000.ckpt --dataset_name=pascalvoc_2007
```

**Comment by youngwanLEE** [comment]
Could you show your train script?

**Comment by samuel1208** [comment]
@youngwanLEE  I didn't train my own model. I test the model from [here](https://drive.google.com/file/d/0B0qPCUZ-3YwWZlJaRTRRQWRFYXM/view). Which was evaluate in the ReadMe with 0.778 mAP

**Comment by miraclebiu** [comment]
@samuel1208  Can you help me to know how to get the map number ? When I run this script, I get these information:
AP_VOC07/mAP[5.3689601551968713e-05]
AP_VOC12/mAP[1.2254785089444698e-05]
there are so weird.

**Comment by xaibeing** [comment]
@miraclebiu I have also got the very pool mAP, but I figure out it's because I make a mistake setting parameter "--checkpoint_path=./checkpoints/". Check these parameters carefully. The  correct command I use is:
`python eval_ssd_network.py --eval_dir=./logs/ --dataset_dir=./data/VOC2007/tfrecords/ --dataset_name=pascalvoc_2007 --dataset_split_name=train --model_name=ssd_300_vgg --checkpoint_path=./checkpoints/ssd_300_vgg.ckpt --batch_size=1`

@samuel1208 I got these result using the pretrained model in your comment: 
AP_VOC07/mAP[0.75743894621489116] AP_VOC12/mAP[0.78580560176307424]

btw, using checkpoints already in this project (checkpoints/ssd_300_vgg.ckpt), I got result:
AP_VOC07/mAP[0.820180232902144] AP_VOC12/mAP[0.85556363127456159]

thanks @balancap sharing this amazing work!

**Comment by miraclebiu** [comment]
@xaibeing   in your scritp  --dataset_split_name=train ? Why? Shouldn't it be test?

**Comment by samuel1208** [comment]
@xaibeing why it's train split not test

**Comment by walterwang** [comment]
@xaibeing is the ssd_300_vgg.ckpt a folder file or the name of a checkpoint file, if so what do you do with the index file?

@samuel1208 @miraclebiu ok so I figured it out: when you extract from the zip file, do not include the folder

This is my configuration: 
#!/usr/bin/env bash
EVAL_DIR=./logs/
CHECKPOINT_PATH=./checkpoints/ssd_300_vgg.ckpt
DATASET_DIR=./tfrecords/
python eval_ssd_network.py \
    --eval_dir=${EVAL_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --dataset_split_name=test \
    --model_name=ssd_300_vgg \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --batch_size=1

This is what my checkpoint direcotry look like :
~/Documents/SSD-Tensorflow/checkpoints$ ls
ssd_300_vgg.ckpt.data-00000-of-00001  ssd_300_vgg.ckpt.index

**Comment by huunhan312** [comment]
I run the available weight in directory checkpoints named "ssd_300_vgg.ckpt". The result on VOC07 Test are
AP_VOC07/mAP[0.82107575232572283]
AP_VOC12/mAP[0.8562692165386977]

However, the result with VGG_VOC0712_SSD_300x300_iter_120000.ckpt are
AP_VOC07/mAP[0.59928035643648037]
AP_VOC12/mAP[0.60921371052809159]

The result with VGG_VOC0712_SSD_300x300_ft_iter_120000.ckpt are
 AP_VOC07/mAP[0.74313215403145938]
AP_VOC12/mAP[0.76659692689602577]

The default checkpoint for ssd_300_vgg is good. But the given weight factor reported in the README cannot reach accuracy as their report. 

Did anyone solve the problem?

**Comment by MerlinBu** [comment]
> I run the available weight in directory checkpoints named "ssd_300_vgg.ckpt". The result on VOC07 Test are
> AP_VOC07/mAP[0.82107575232572283]
> AP_VOC12/mAP[0.8562692165386977]
> 
> However, the result with VGG_VOC0712_SSD_300x300_iter_120000.ckpt are
> AP_VOC07/mAP[0.59928035643648037]
> AP_VOC12/mAP[0.60921371052809159]
> 
> The result with VGG_VOC0712_SSD_300x300_ft_iter_120000.ckpt are
> AP_VOC07/mAP[0.74313215403145938]
> AP_VOC12/mAP[0.76659692689602577]
> 
> The default checkpoint for ssd_300_vgg is good. But the given weight factor reported in the README cannot reach accuracy as their report.
> 
> Did anyone solve the problem?

I get same result, did you solve it?

**Comment by xbcReal** [comment]
@MerlinBu I have totally same result as you and do you know what's wrong with this code?

**Comment by MerlinBu** [comment]
> @MerlinBu I have totally same result as you and do you know what's wrong with this code?

他这个结果貌似是在验证集而不是测试集上跑的。。。。

**Comment by MerlinBu** [comment]
@xbcReal 这貌似是验证集的结果，不是测试集。。。
